packages/zosfiles/src/cli/hDelete/HDelete.definition.ts

```diff
@@ -0,0 +1,32 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition } from "@zowe/imperative";
+import { DsDefinition } from "./ds/Ds.definition";
+
+import i18nTypings from "../-strings-/en";
+
+// Does not use the import in anticipation of some internationalization work to be done later.
+const { DESCRIPTION } = (require("../-strings-/en").default as typeof i18nTypings).HDELETE;
+
+/**
+ * hDelete group definition containing its description and children
+ * @type {ICommandDefinition}
+ */
+export const HDeleteDefinition: ICommandDefinition = {
+    name: "delete",
```

Comment:
Would it be better to change this to `hDelete`, as the `Delete Data Set` command also uses `delete`. And therefore should `migrate` and `recall` be changed to `hMigrate` and `hRecall` for consistancy?

Reply:
I believe that is why the integration test is failing^

Reply:
I do not remember all the discussions around the naming of the other two commands, however if people sitll want to use the other commands as hRecall or hMigrate, they can (since they are aliases)

For the delete, why not reuse the same delete command?

Reply:
Yeah that's a good idea, Ill do that now